package.json

```diff
@@ -26,7 +26,6 @@
     "@babel/preset-env": "^7.16.4",
     "@gliff-ai/eslint-config": "^0.2.0",
     "@gliff-ai/jest-browserstack-automate": "^5.0.0",
-    "@gliff-ai/style": "^2.8.0",
```

Comment:
I think you still want the dev dependency for local development, it just needs to match the peer dependency? Unless @cooper667 knows of a nice way where we define the version once (as peer) but can install peer deps under dev?

Reply:
Yes to having it in dev AND peer. There's no nice way to have both really.